CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2022-04-13 HFI Calc - Refactor Request & Response Structure [#1555](https://github.com/bcgov/wps/issues/1555)
+
+### Refactor
+
+- **hfi calculator** Changed HFI request and result structure, remove selected_station_code_ids and made request/response structure more strict.
```

Comment:
for my own clarification - are we adding everything to the changelog, or just the changes that users will see?

Reply:
I don't have a good answer. I feel like we always have the commit logs - so we don't need this in there. On the other hand, if we're going to autogenerate a commit log, it will end up having this in.
I don't know. I'm not going to say no. I'm not going to say yes.

Reply:
I'm hoping that we can filter the types of changes we include in the autogenerated changelog, e.g. only include `feat` commit logs, and we'd tag refactors differently using the spec'd `chore` or something if possible.